src/plots/barPlot.ts

```diff
@@ -298,6 +299,24 @@ export module Plots {
       return Plottable.Utils.DOM.intersectsBBox(xRange, yRange, barBBox);
     }
 
+    protected _datumVisibleOnPlot(pixelPoint: Point, datum: any, index: number, dataset: Dataset) {
```

Comment:
Why is the order like so?  Right now we have an order of
`datum`, `pixelPoint`, `index`, `dataset` consistently